codegen/templates/workflow_mock.tmpl

```diff
@@ -31,7 +31,7 @@ import (
 	{{end -}}
 	{{end -}}
 	{{end -}}
-	{{$instance.PackageInfo.PackageAlias}} "{{$instance.PackageInfo.PackagePath}}"
+	{{$instance.PackageInfo.PackageAlias}} "{{$espec.WorkflowImportPath}}"
```

Comment:
nit: This would be confusing because `PackageAlias` usually always refers to the import path for `PackagePath`. Maybe call it `{{$instance.PackageInfo.PackageAlias}}workflow` to disambiguate it?

Reply:
i am scared of making a larger change so i will do it.... later